### ISSUE TITLE
Conjunto de treinamento e de teste

### ISSUE THREAD

**Issue by RafaellaJunqueira** [deeplearningunb/projetoDL2021#2]
- [x] Instalar dependências
- [x] Criar conjunto de treinamento (Training set)
- [x] Criar conjunto de teste (Test set)
